backend/model/model.go

```diff
@@ -1961,10 +1961,12 @@ type Alert struct {
 	Default           bool                `gorm:"default:false"` // alert created during setup flow
 
 	// fields for threshold alert
-	BelowThreshold    *bool
-	ThresholdValue    *float64
-	ThresholdWindow   *int
-	ThresholdCooldown *int
+	BelowThreshold     *bool
+	ThresholdValue     *float64
+	ThresholdWindow    *int
+	ThresholdCooldown  *int
+	ThresholdType      modelInputs.ThresholdType
+	ThresholdCondition modelInputs.ThresholdCondition
```

Comment:
Not sure whats going on, but was seeing these errors when loadin/saving an alert. The code looks good to me, and confirmed in postgres that these columns exists although they were null. Do we need a default here or to migrate data?

```
Cannot query field "threshold_type" on type "Alert". Did you mean "threshold_value" or "threshold_window"?
Cannot query field "threshold_condition" on type "Alert". Did you mean "threshold_window" or "threshold_cooldown"?
```

Reply:
those look like graphql backend errors and I see those fields in the `generated.go` file so I'm not sure what's going on, I was able to load/save alerts locally. are you getting any errors running the backend locally?

Reply:
was originally planning on requiring a migration (see sql script in PR description), but just added some code to handle this as well for on-prem / hobby customers